src/core/Input/InputField/styles.ts

```diff
@@ -0,0 +1,59 @@
+import { makeStyles, Theme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  disabledInput: {
+    border: '0.0.625rem solid',
+    background: theme.palette.background.paper,
+    color: theme.palette.text.disabled,
+    '& fieldset': {
+      borderColor: theme.palette.border.main,
+    },
+  },
+  primaryInput: {
+    border: '0.0.625rem solid',
+    background: theme.palette.background.paper,
+    '& fieldset': {
+      borderColor: theme.palette.border.main,
+    },
+    '&:hover': {
+      '& fieldset': {
+        borderColor: `${theme.palette.highlight} !important`,
+      },
+    },
+  },
```

Comment:
Keep this at the top and rename it to `root`. Also add text field related css to it, like borderRadius, width, etc